pkg/backup/item_backupper.go

```diff
@@ -341,12 +348,16 @@ func (ib *itemBackupper) executeActions(
 				return nil, errors.WithStack(err)
 			}
 
-			if _, err = ib.backupItem(log, item, gvr.GroupResource(), gvr); err != nil {
+			if _, err = ib.backupItem(log, item, gvr.GroupResource(), gvr,
+				u.GetAnnotations()[mustIncludeAdditionalItemAnnotation] == "true"); err != nil {
```

Comment:
Rather than calling `u.GetAnnotations()` for each returned additional item, why not add this after line 324 above:
```
mustInclude := u..GetAnnotations()[mustIncludeAdditionalItemAnnotation] == "true")
```
and just use this bool var here to pass into `backupItem`

Reply:
@sseago Thanks.
The goal for this draft is to demonstrate the idea, if you are not against the idea, I will refine it and set it to ready for review.